Tests/Command/ConsumeCommandTest.php

```diff
@@ -32,14 +55,14 @@ public function setMockConsumer($expirationCount)
             ->willReturn(true);
 
         self::$kernel->getContainer()->set('smartesb.consumers.queue', $this->mockConsumer);
-        self::$kernel->getContainer()->set('smartesb.consumers.async_queue', $this->mockConsumer);
-        self::$kernel->getContainer()->set('doctrine', $this->createMock(RegistryInterface::class));
     }
 
     public function testExecuteWithKillAfter()
     {
+        $this->setMockQueueDriver();
         $this->setMockConsumer(self::NB_MESSAGES);
 
+        self::$kernel->getContainer();
```

Comment:
Which is the reason of calling `getContainer()` here?

Reply:
It was some rest of code to test. I removed this line.